python/packages/ai/teams/ai/ai_options.py

```diff
@@ -40,3 +40,5 @@ class AIOptions(Generic[StateT]):
     Optional. If true, the AI system will enable the feedback loop in Teams that
     allows a user to give thumbs up or down to a response.
     """
+
+    feedback_loop_type: Union[Literal["default"], Literal["custom"]] = "default"
```

Comment:
we also need to propagate the type for streaming scenarios - I did it for JS in this [PR](
https://github.com/microsoft/teams-ai/pull/2185/files#diff-9f02d3e841517a187ac805b1d4116026d51d62aae88469aa5444d6ea3675beceR339)